packages/manifest-schemas/formats/web3api.build/0.0.1-prealpha.1.json

```diff
@@ -7,28 +7,34 @@
   ],
   "properties": {
     "format": {
+      "description": "Polywrap build YAML format version.",
```

Comment:
Can you also replace the YAML here with manifest?

Reply:
Type here is string. Do we accept json as well as main manifest file?

Reply:
Yes we do!

Reply:
Do we accept json as main manifest file?
I am assuming, we can keep YAML web3api/0.0.1/prealpha.1.json?

Reply:
updated web3api.build/0.0.1-prealpha.1.json and web3api.build/0.0.1-prealpha.2.json